src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -124,4 +124,5 @@ BasicDateInput.defaultProps = {
   isDirty: false,
   errorType: '',
   errorMessages: {},
+  // value: new Date(undefined),
```

Comment:
Commented code to remove :slightly_smiling_face:

Reply:
Nice catch, thanks! :slightly_smiling_face: